test/cppsuite/tests/cursor_bound_01.cpp

```diff
@@ -578,30 +607,33 @@ class cursor_bound_01 : public test {
         logger::log_msg(
           LOG_INFO, type_string(tc->type) + " thread {" + std::to_string(tc->id) + "} commencing.");
 
-        bound lower_bound, upper_bound;
         std::map<uint64_t, scoped_cursor> cursors;
-
+        std::map<uint64_t, std::pair<bound, bound>> bounds;
```

Comment:
Worth a comment to describe what these maps are and how they relate to one another.